framework/src/systems/AuxiliarySystem.C

```diff
@@ -644,12 +644,21 @@ AuxiliarySystem::computeElementalVarsHelper(
     {
       ConstElemRange & range = *_mesh.getActiveLocalElementRange();
       ComputeElemAuxVarsThread<AuxKernelType> eavt(_fe_problem, warehouse, vars, true);
-      Threads::parallel_reduce(range, eavt);
-
-      solution().close();
-      _sys.update();
+      try
```

Comment:
I certainly seem to get many of my exception problems with the aux systems, so I'm glad to see this. @lindsayad can you double check this type of protection exists in all the other systems? Perhaps it's not really applicable to all, but off hand I'm thinking materials, dampers, executioners, constraints...dunno, you get the idea.

Reply:
nodal aux?